tests/integration/test_charm.py

```diff
@@ -30,22 +31,54 @@
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.skip(reason="error Waiting for pebble will be addressed in a followup PR")
-async def test_synapse_from_refresh_is_up(
-    synapse_refresh_app: Application,
+async def test_synapse_with_mjolnir_from_refresh_is_up(
+    ops_test: OpsTest,
+    model: Model,
+    synapse_charmhub_app: Application,
     get_unit_ips: typing.Callable[[str], typing.Awaitable[tuple[str, ...]]],
+    synapse_charm: str,
+    synapse_image: str,
+    synapse_nginx_image: str,
 ):
     """
-    arrange: build and deploy the Synapse charm.
-    act: send a request to the Synapse application managed by the Synapse charm.
-    assert: the Synapse application should return a correct response.
+    arrange: build and deploy the Synapse charm from charmhub and enable Mjolnir.
```

Comment:
I think we don't deploy the charm from charm hub but rather the locally built one, if that's the case then perhaps you can update this line.

Reply:
Not really, the charm is deployed from Charmhub in the fixture (see https://github.com/canonical/synapse-operator/blob/be4d06e64a02c073575f863e4622fc6036cfb239/tests/integration/conftest.py#L139-L146), and then refreshed with the local one.